src/fakeredis_instance.erl

```diff
@@ -181,9 +181,15 @@ handle_request([<<"ASKING">>], State) ->
     %% The client_asking flag is handled by handle_data/2.
     encode_and_send(ok, State);
 
+handle_request([<<"EVALSHA">> | _Tail], State) ->
+    %% No script handling yet in fakeredis_cluster
+    %% Reply like Redis when there is no script available
+    encode_and_send({error, <<"NOSCRIPT No matching script. Please use EVAL.">>},
+                    State);
```

Comment:
Does eredis_cluster match the exact error message or only the NOSCRIPT prefix? If it only cares about the prefix (which is what I think would make sense) perhaps the error can instead say "NOSCRIPT No matching script because EVALSHA is not implemented. Please use EVAL." or so. Not very important thouh.

Reply:
The text is a bit weird, but its what redis spits out when the script is not available

Reply:
eredis_cluster just cares about the "NOSCRIPT " part of it

Reply:
I just though it can be confusing that we return this even if EVAL has already been used with the same script. I realize now that EVAL will always fail also, so this is fine.